integration/tests/conftest.py

```diff
@@ -60,6 +60,7 @@ def record_test_metric(request):
                 'host': socket.gethostname(),
                 'user': getpass.getuser(),
                 'run-id': os.getenv('TEST_METRICS_RUN_ID', None),
+                'run-description': os.getenv('TEST_METRICS_RUN_DESCRIPTION', 'open source integration tests'),
```

Comment:
why "open source integration tests" and not just "open source tests"

Reply:
Because this only covers the integration tests (the unit tests are in Clojure)